uni-snd/src/web_snd.rs

```diff
@@ -39,12 +52,35 @@ impl<T> SoundDriver<T> {
             err,
         }
     }
+    // -1 => game paused
+    // >0 => pause duration
+    fn get_pause_status(&self) -> f64 {
```

Comment:
Maybe using Option here ? e.g
```
fn get_pause_duration(&self) -> Option<f64>
```

Reply:
indeed... I'm still not used to idiomatic rust :)